File: lib/model/ui/world_speed.dart
enum WorldSpeed {
  normal,
  fast,
  faster,
}

extension WorldSpeedExtension on WorldSpeed {
  int get speed {
    switch (this) {
      case WorldSpeed.normal:
        return 1;
      case WorldSpeed.fast:
        return 2;
      case WorldSpeed.faster:
        return 3;
    }
  }
}

File: ios/Runner/Processor/model/world/WorldSpeed.swift
//
//  WorldSpeed.swift
//  Runner
//
//  Created by Serhii Rakovskyi on 17.08.2022.
//

import Foundation

enum WorldSpeed :Codable {
    case normal
    case fast
    case faster
    
    var description : String {
        switch self {
        case .normal:
            return "normal"
        case .fast:
            return "fast"
        case .faster:
            return "faster"
        }
    }
}
